docs/reference/glossary.md

```diff
@@ -72,7 +72,7 @@ A Docker container consists of
 - Execution environment
 - A standard set of instructions
 
-The concept is borrowed from Shipping Containers, which define a standard to ship
+The concept is borrowed from Shipping Containers, which defines a standard to ship
```

Comment:
Sorry, this is not correct. "Shipping Containers" is plural, and "they define a standard"